Templates/Integrations/AnalyticsAndSIEM/AnalyticsAndSIEM.py

```diff
@@ -0,0 +1,488 @@
+        context = {
+            f'{INTEGRATION_CONTEXT_NAME}.Event(val.ID && val.ID === obj.ID)': context_entry
+        }
+        human_readable = tableToMarkdown(title, context_entry)
+        return human_readable, context, raw_response
+    else:
+        raise DemistoException(f'{INTEGRATION_NAME} - Could not update event `{event_id}`')
+
+
+def create_event(client: Client, args: Dict) -> Tuple[str, Dict, Dict]:
+    """Creates a new event and return outputs in Demisto's format
+
+    Args:
+        client: Client object with request
+        args: Usually demisto.args()
+
+    Returns:
+        Outputs
+    """
+    # Get arguments from user
+    description = args.get('description', '')
+    assignee = argToList(demisto.args().get('assignee', ''))
+    # Make request and get raw response
+    raw_response = client.create_event_request(description, assignee)
+    events = raw_response.get('event')
+    # Parse response into context & content entries
+    if events:
+        event = events[0]
+        event_id: str = event.get('eventId', '')
+        title = f'{INTEGRATION_NAME} - Event `{event_id}` has been created.'
+        context_entry = build_context(event)
+        context = {
+            f'{INTEGRATION_CONTEXT_NAME}.Event(val.ID && val.ID === obj.ID)': context_entry
+        }
+        human_readable = tableToMarkdown(title, context_entry)
+        return human_readable, context, raw_response
+    else:
+        raise DemistoException(f'{INTEGRATION_NAME} - Could not create new event.')
+
+
+def query(client: Client, args: Dict) -> Tuple[str, Dict, Dict]:
+    """Search for event by given args
+
+    Args:
+        client: Client object with request
+        args: Usually demisto.args()
+
+    Returns:
+        Outputs
+    """
+    # Get arguments from user
+    query_dict = assign_params(
+        eventId=argToList(args.get('event_id')),
+        fromTime=args.get('event_created_date_after'),
+        toTime=args.get('event_created_date_before'),
+        assignee=argToList(args.get('assignee')),
+        isActive=args.get('is_active') == 'true' if args.get('is_active') else None
+    )
+    # Make request and get raw response
+    raw_response = client.query_request(**query_dict)
+    events = raw_response.get('event')
+    # Parse response into context & content entries
+    if events:
+        title = f'{INTEGRATION_NAME} - Results for given query'
+        context_entry = build_context(events)
+        context = {
+            f'{INTEGRATION_CONTEXT_NAME}.Event(val.ID && val.ID === obj.ID)': context_entry
+        }
+        human_readable = tableToMarkdown(title, context_entry)
+        return human_readable, context, raw_response
+    else:
+        return f'{INTEGRATION_NAME} - Could not find any results for given query', {}, {}
+
+
+''' COMMANDS MANAGER / SWITCH PANEL '''
+
+
+def main():  # pragma: no cover
+    params = demisto.params()
+    base_url = f"{params.get('url', '').rstrip('/')}'/api/v2/'"
```

Comment:
I dont think we need to provide default value of `''` url. We should return error in that case
use `urljoin(params.get('url'), '/api/v2/')`